src/OrangePaymentSlipData.php

```diff
@@ -100,6 +108,7 @@ public function setWithBankingCustomerId($withBankingCustomerId = true)
 
         if ($withBankingCustomerId === false) {
             $this->bankingCustomerId = '';
+            $this->bankingCustomerIdLength = 0;
```

Comment:
I don't agree, if `$this->bankingCustomerId = '';`, `$this->bankingCustomerIdLength = 0;` must be set to `0` and not to `null`.

Futhermore, the specified type is integer, so it should never be null.